plugins/org.locationtech.udig.libs/build.properties

```diff
@@ -166,7 +165,6 @@ bin.includes = .,\
                lib/ion-java-1.0.2.jar,\
                lib/istack-commons-runtime-3.0.7.jar,\
                lib/jackson-annotations-2.7.9.jar,\
-               lib/jackson-core-2.9.9.jar,\

```

Comment:
just have to check, why it was not loaded by build with pom-libs.xml, same for 
* jgridshift-1.1.jar and 
* axis-1.4.jar

During the build I saw some Xerces errors, try to reproduce it and post it then